pod.go

```diff
@@ -389,6 +389,20 @@ func (p *Pod) Annotations(key string) (string, error) {
 	return value, nil
 }
 
+// SetAnnotations sets or adds an annotations
+func (p *Pod) SetAnnotations(annotations map[string]string) error {
+	for k, v := range annotations {
+		p.config.Annotations[k] = v
```

Comment:
Once we lock `Annotations`, we also need to provide a `GetAnnotation()` getter and unexport `Annotations`.